src/backend/storeManagers/nile/library.ts

```diff
@@ -66,7 +66,8 @@ function loadGamesInAccount() {
     } = productDetail
 
     const info = installedGames.get(product.id)
-
+    // Create save folder name like nile
+    const safeFolderName = removeSpecialcharacters(title ?? '')
```

Comment:
Is it really a good idea to use `''` as a fallback here? I can't imagine that creating a folder with that name would work

Reply:
since `folder_name` for nile games is only used when deleting canceled installation `''` will be `false` therefore it won't cause any trouble

Reply:
Also I just confirmed the case when title was undefined is resolved, so this is even less likely to happen